ResearchKit/Common/ORKTask.h

```diff
@@ -167,6 +167,13 @@ ORK_AVAILABLE_DECL
 - (ORKTaskProgress)progressOfCurrentStep:(ORKStep *)step withResult:(ORKTaskResult *)result;
 
 /**
+ Validates the task parameters.
+ 
+ This method should be used to check that all the task parameters are correct. E.g., `ORKOrderedTask` makes sure that all its step identifiers are unique. It should throw an exception if validation fails.
```

Comment:
Thanks for the documentation comment feedback, I just rewrote it.

However, I feel confortable about throwing an exception from this method instead of returning a boolean/error. What would you do with the returned error other than throwing an exception? Also, `ORKAnswerFormat`'s `validationParameters` does the same (in addition to `ORKStep`'s).

If you still want to do that nevertheless, let me know and I can create a follow-on issue.